packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -1864,6 +1864,7 @@
   "theme-custom-css": "Custom CSS",
   "theme-font-body-font-family": "Body Font Family",
   "There_are_no_agents_added_to_this_department_yet": "There are no agents added to this department yet.",
+  "There_are_no_applications": "There are no OAuth Applications added yet.",
```

Comment:
I wonder, would it make more sense to have it worded: "No oAuth Applications have been added yet."? I'm not 100% sure :thinking:

Reply:
Definitely makes more sense. I’m gonna change that later. Should I change the variable name as well?

Reply:
I don't think so, should be fine with how it currently is. Thanks!